docs/_ingest/http.md

```diff
@@ -21,6 +21,36 @@ The most common types to use are `application/n-quads`, `text/turtle` and `appli
 
 For more details, please refer to the Swagger API under the `base` definition.
 
+## Version objects, state objects and version creation
+
+Currently, an event stream can ingest version objects by default. A version object describe the state of a specific
+version of a resource at a specific timestamp.
+
+However, an event stream can be configured in such a way that it can ingest state objects and that version objects can
```

Comment:
Would cut the sentence in two.  ".. that it can ingest state objects." "Once retrieving the event stream and its fragments, the members will be presented as version objects."